spec/controllers/tasuku/taskables/questions/answers_controller_spec.rb

```diff
@@ -32,6 +32,20 @@ module Tasuku
 
           expect(question.answers.count).to eq 1
         end
+
+        context "with submitted answer" do
+          let(:user)     { create :user }
+          let(:question) { create :question_with_options }
+          let(:vote) { create :question_vote, option: question.options.first }
+          let(:question_answer) { create :question_answer, author: user, votes: [vote] }
+          let(:update_params) { { question_id: question.id, id: question_answer.id  } }
+
+          it 'updates a existing answer' do
```

Comment:
In spec description you expect it to update something, but you it doesn't really check it in spec itself